### ISSUE TITLE
Underrated

### ISSUE THREAD

**Issue by rchavezj** [vvmnnnkv/libtorchjs#3]
Not gonna lie, this repo is hella underrated. PyTorch is an excellent machine learning framework, c++ is hella fast, and the community for production it is hella small. You can bind pytorch (c++ libtorch) with golang, rust, nodejs so I'm quite impressed with ur readme.md explanation.

**Comment by vvmnnnkv** [comment]
@rchavezj thanks!
If you have usage examples of this lib, I can add them to readme :)

**Comment by rchavezj** [comment]
I recently got a job as a react developer :) I have custom pytorch algorithms I plan on displaying on react for the UI side. I look forward using ur NPM for my side healthcare projects

**Comment by rchavezj** [comment]
I'm part of the biggest online A.I. community so I hope it okay if I post about ur NPM on the page

**Comment by eugeneware** [comment]
I completely agree. This is really excellent work. I started writing my own wrappers, and saw this, and was like - this is much better! Keep up the amazing work!
